package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hubot-hue-meeting",
   "description": "Control your Hue lights for meeting mode.",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Don't bump the version here -- I've got a `Grunt` command I run to publish it to NPM.